src/migrations/versions/c9759675b5d3_add_user_login_and_password_reset_dates.py

```diff
@@ -0,0 +1,31 @@
+"""add user login and password reset dates
+
+Revision ID: c9759675b5d3
+Revises: 906d7198df28
+Create Date: 2022-04-14 13:16:58.940571
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "c9759675b5d3"
+down_revision = "906d7198df28"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "fidesopsuser",
+        sa.Column("last_login_at", sa.Date(), nullable=True),
+    )
+    op.add_column(
+        "fidesopsuser", sa.Column("password_reset_at", sa.Date(), nullable=True)
+    )
+
```

Comment:
Your models have these variables as DateTime type, but the migration adds them as Dates.  I'd change this migration to DateTime type, so we record more detail.